src/api/users/src/models/celebrateSchema.js

```diff
@@ -1,5 +1,13 @@
 const { celebrate, Joi, Segments } = require('celebrate');
 
+const validatePagingParams = () =>
+  celebrate({
+    [Segments.PARAMS]: {
+      per_page: Joi.number(),
```

Comment:
Make your validator do more work for you:

```js
per_page: Joi.number().integer().min(1).max(100),
page: Joi.number().integer().min(1),
```

Reply:
Awesome, thanks so much for this.